environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -205,4 +205,18 @@ SimpleEnvironmentResponses deleteMultipleByCrns(Set<String> crns,
     @ApiOperation(value = EnvironmentOpDescription.CLI_COMMAND, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
             nickname = "getCreateEnvironmentForCli")
     Object getCreateEnvironmentForCli(@NotNull @Valid EnvironmentRequest environmentRequest);
+
+    @POST
+    @Path("/crn/{crn}/update_config")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.UPDATE_CONFIG_BY_CRN, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
+        nickname = "updateConfigsInEnvironmentByCrnV1")
+    void updateConfigsInEnvironmentByCrn(@PathParam("crn") String crn);
+
+    @POST
+    @Path("/name/{name}/update_config")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.UPDATE_CONFIG_BY_NAME, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
```

Comment:
Just have it in there to maintain consistency of how Env service functions are defined.   I'm 50/50 on it being there.  The cluster name reuse is limited by tenant anyways.   But in this case since we are removing auth checks and it won't have an x-cdp-actor-crn it might be best to remove it.

Reply:
Removed the call